configure.ac

```diff
@@ -42,6 +42,9 @@ AC_ARG_ENABLE(pthread,
 AC_ARG_ENABLE(ntopng,
               AC_HELP_STRING([--enable-ntopng], [enable flow sending to ntopng with zeromq (default NO)]),
               [ntopng=yes],[ntopng=no])
+AC_ARG_ENABLE(ifname,
+              AC_HELP_STRING([--enable-ifname], [enable flow reporting iface name in normal data of v9 (default NO)]),
```

Comment:
This option affects not only v9 but also IPFIX exporting. You should write "enable flow reporting iface name in normal data of v9 and IPFIX".

Reply:
changed in this commit
https://github.com/irino/softflowd/pull/33